beetsplug/discogs.py

```diff
@@ -56,7 +60,9 @@
     ConnectionError,
     socket.error,
     http.client.HTTPException,
+    HTTPError,
```

Comment:
Unneeded - `HTTPError` inherits from `DiscogsAPIError` which is already present in the list.